content/en/news/releases/1.25.x/announcing-1.25/change-notes/index.md

```diff
@@ -0,0 +1,263 @@
+
+- **Fixed** failed to patch managed gateway/waypoint deployment during upgrade to 1.24.
+  ([Issue #54145](https://github.com/istio/istio/issues/54145))
+
+- **Fixed** non-default revisions controlling gateways lacking istio.io/rev labels.
+  ([Issue #54280](https://github.com/istio/istio/issues/54280))
+
+- **Fixed** the wording of the status message when L7 rules are present in an AuthorizationPolicy which is bound to ztunnel to be clearer.
+  ([Issue #54334](https://github.com/istio/istio/issues/54334))
+
+- **Fixed** a bug where request mirror filter incorrectly computing the percentage.
+  ([Issue #54357](https://github.com/istio/istio/issues/54357))
+
+- **Fixed** an issue where using a tag in the istio.io/rev label on a gateway causes the gateway to be improperly programmed, and to lack status.
+  ([Issue #54458](https://github.com/istio/istio/issues/54458))
+
+- **Fixed** issue where out-of-order ztunnel disconnects could put `istio-cni` in a state where it believes it has no connections.
+  ([Issue #54544](https://github.com/istio/istio/issues/54544)),([Issue #53843](https://github.com/istio/istio/issues/53843))
+
+- **Fixed** excessive iptables info-level log entries for rule checks and deletions.
+Detailed logging can be re-enabled by switching to debug-level logs if necessary.
+  ([Issue #54644](https://github.com/istio/istio/issues/54644))
+
+- **Fixed** an issue causing `ExternalName` services failing to resolve when using Ambient mode and DNS proxying.
+
+- **Fixed** an issue causing configuration to be rejected when there is a partial overlap between IP addresses across multiple services.
+For example, a Service with `[IP-A]` and one with `[IP-B, IP-A]`.  ([Issue #52847](https://github.com/istio/istio/issues/52847))
+
+- **Fixed** an issue causing VirtualService header name validation to reject valid header names.
+
+- **Fixed** an issue when upgrading waypoint proxies from Istio 1.23.x to Istio 1.24.x.
+  ([Issue #53883](https://github.com/istio/istio/issues/53883))
+
+## Security
+
+- **Added** the `DAC_OVERRIDE` capability to the `istio-cni-node` DaemonSet. This fixes issues when running in environments
+where certain files are owned by non-root users.
+Note: prior to Istio 1.24, the `istio-cni-node` ran as `privileged`. Istio 1.24 removed this, but removed some required
+privileges which are now added back. Relatively to Istio 1.23, `istio-cni-node` still has fewer privileges than it does with this change.
+
+- **Added** unconfined AppArmor annotation to the istio-cni-node DaemonSet to avoid conflicts with
+AppArmor profiles which block certain privileged pod capabilities. Previously, AppArmor
+(when enabled) was bypassed for the istio-cni-node DaemonSet since privileged was set to true
+in the SecurityContext. This change ensures that the AppArmor profile is set to unconfined
+for the istio-cni-node DaemonSet.
```

Comment:
```suggestion
- **Added** unconfined AppArmor annotation to the `istio-cni-node` `DaemonSet` to avoid conflicts with
  AppArmor profiles which block certain privileged pod capabilities. Previously, AppArmor
  (when enabled) was bypassed for the `istio-cni-node` `DaemonSet` since privileged was set to true
  in the `SecurityContext`. This change ensures that the AppArmor profile is set to unconfined
  for the `istio-cni-node` `DaemonSet`.
```